src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: $card-width;
+
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__img {
+    display: block;
+    max-height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #{$third-color};
```

You don't need to use interpolation
```suggestion
    color: $third-color;
```

---

ok, I will change.